test/performance/scheduler/default_rangespec.yaml

```diff
@@ -25,5 +25,5 @@ wlClassesMaxAvgTimeToAdmissionMs:
   # Average value 76768 (+/- 2%), setting at +20%
   medium: 90_000
 
-  # Average value 215468 (+/- 2%), setting at +5%
-  small: 227_000
+  # Average value 215468 (+/- 2%), setting at +5% + 3s accounting for observed failures
+  small: 230_000
```

Comment:
Is this enough?
Actually, we observed the 231068ms here: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/directory/pull-kueue-test-scheduling-perf-main/1795196252221083648

Reply:
Oh I didn't know about the other failure, then maybe 233_000, to have some buffer?